make

```diff
@@ -78,6 +87,22 @@ function main() {
     todo)
       grep -E 'TODO:|FIXME:' **/*.[ch]
       ;;
+    docker-build)
+      shift
+      # Cygwin is its own special case; Linux/Mac/WSL can be handled the same way
```

Comment:
This should probably be an external script instead of inline.  The 'make' script is mostly just a dispatcher, and non-trivial amounts of code should be in other files.

Reply:
Done.